src/Template/Error/error400.twig

```diff
@@ -0,0 +1,38 @@
+{% element 'modules_top_menu' %}
+
+{% element 'left_menu' %}
+
+<div class="head">
+    <h1>{{ __('An Internal Error Has Occurred') }}</h1>
+</div>
+
+<div class="main">
+
+    <p class="error">
+        {{ message }}
+    </p>
+
+    {% if config('debug') %}
+
+        {% if error.queryString %}
```

Comment:
don't know if we will ever get an SQL error, surely not now...
I would simply remove or comment out these lines - here and in `error500.twig`